deploy/configure_tls_trust.sh

```
@@ -0,0 +1,45 @@
+  case "$ssl_choice" in
+    2)
+      echo "Using enable_insecure for development/self-signed environments."
+      TRUST_HELM_ARGS='--set enable_insecure=true'
+      ;;
+    3)
+      local ca_secret ca_file
+      read -p "Name of the Secret containing PEM-encoded CA cert(s) [company-internal-ca]: " ca_secret
+      ca_secret=${ca_secret:-company-internal-ca}
+
+      if ! kubectl -n "$ns" get secret "$ca_secret" >/dev/null 2>&1; then
+        read -p "Secret '$ca_secret' not found. Provide path to PEM CA bundle file: " ca_file
```

Shellcheck (0.11.0)

[info] read without -r will mangle backslashes.

(SC2162)